lib/Entities/FormEntity.php

```diff
@@ -102,4 +102,17 @@ public function setStyles($styles) {
     $this->styles = $styles;
   }
 
+  public function jsonSerialize() {
+    return [
+      'id' => $this->getId(),
+      'name' => $this->getName(),
+      'body' => $this->getBody(),
+      'settings' => $this->getSettings(),
+      'styles' => $this->getStyles(),
+      'created_at' => $this->getCreatedAt(),
+      'updated_at' => $this->getUpdatedAt(),
+      'deleted_at' => $this->getDeletedAt(),
+    ];
+  }
+
```

Comment:
We're not doing this in other places yet and I'm just thinking whether it's a good idea. Self-serialization of entities is always a problem since different use cases require different data but I guess there's no harm in this case.

But maybe instead of implementing `\JsonSerializable` you could just and special method such as `toArray()`? It has the advantage of being easily searchable if one day we want to know in how many places this is used.